src/riak_pb_ts_codec.erl

```diff
@@ -243,6 +245,44 @@ decode_cells([#tscell{varchar_value = undefined,
     %% TODO: represent null cells by something other than an empty list. emptyTsCell atom maybe?
     decode_cells(T, [[] | Acc]).
 
+
+
+%% Copied and modified from riak_kv_pb_coverage:convert_list. Would
+%% be nice to collapse them back together, probably with a closure,
+%% but time and effort.
+encode_cover_list(Entries) ->
```

Comment:
We generally try to make sure exported functions have specs. Internal ones are less important.